site/docs/releases.md

```diff
@@ -56,8 +56,46 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
   ...
 </dependencies>
 ```
+## 0.12.0 Release Notes
 
-## 0.11.1 release notes
+Apache Iceberg 0.12.0 was released on {{ DATE }}. It consists of {{ NUMCOMMITS }} commits contributed by {{ NUMCOMMITTERS }} over a {{ NUMDAYS }} day period.
+
+High-level features:
+
+* **Core**
```

Comment:
Also added the ability to upgrade to v2 or create a v2 table using the table property `format-verison=2`.

Reply:
Done